rerun_py/rerun_sdk/rerun/blueprint/archetypes/tensor_view_fit_ext.py

```diff
@@ -0,0 +1,26 @@
+from __future__ import annotations
+
+from typing import Any
+
+from ...error_utils import catch_and_log_exceptions
+from .. import components as blueprint_components
+
+
+class TensorViewFitExt:
+    """Extension for [TensorViewFit][rerun.blueprint.archetypes.TensorViewFit]."""
+
+    def __init__(self: Any, scaling: blueprint_components.ViewFitLike | None = None):
```

Comment:
I'm not following why we need an `_ext.py` here. This looks like what I_would expect the default constructor to look like.

Reply:
our default archetype constructors take named arguments only. Not only is that not what I want here, but also it makes the "just pass a ViewFitLike on the view type" not work.